tests/docker-compose.yml

```
@@ -2,8 +2,24 @@ version: '3.1'
 
 services:
 
-  db:
-    image: postgres:9.0
+  postgres-9-0:
+    image: postgres:9.0.22
+    restart: always
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: example
+      POSTGRES_DB: demo
+
+  postgres-9-1:
+    image: postgres:9.1.24
+    restart: always
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: example
+      POSTGRES_DB: demo
+
+  postgres-9-2:
+    image: postgres:13.3
```

I understand that we use the same query for 9.2 *and higher*, but without context it might be confusing to see this or might even look like a typo.

What about changing this to:
```suggestion
  postgres-9-2-onwards:
    image: postgres:13.3
```

or just

```suggestion
  postgres-13:
    image: postgres:13.3
```

---

Nice. I'll go with 9-2-onwards to match the logic of the code